examples/camel-example-ceylon/src/main/ceylon/org/apache/camel/example/run.ceylon

```diff
@@ -0,0 +1,32 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import org.apache.camel.main { CamelMain = Main }
+import org.apache.camel.example { MyRouteBuilder }
+import ceylon.interop.java { createJavaStringArray }
+
+shared void run() {
+    print("\n\n\n\n");
+    print("===============================================");
+    print("Open your web browser on http://localhost:8080");
+    print("Press ctrl+c to stop this example");
+    print("===============================================");
+    print("\n\n\n\n");
+
+    CamelMain main = CamelMain();
+    main.addRouteBuilder(MyRouteBuilder());
+    main.run(createJavaStringArray(process.arguments));
```

Comment:
it's not about just parameters, it's necessary to call main.main(...) method or add System.exit in ceylon code.
Please look at this code as basis https://github.com/apache/camel/blob/241f7bd00ba3f3a2a556ec74de15e73901051ba6/camel-core/src/main/java/org/apache/camel/main/Main.java#L48

Reply:
Thanks that indeed fixed the error on ctrl+c!